src/core_systems/continuous_time_sde.jl

```diff
@@ -0,0 +1,245 @@
+using SciMLBase: SDEProblem, AbstractSDEIntegrator, __init, SDEFunction
+using StochasticDiffEq: SOSRA
+export CoupledSDEs
+
+###########################################################################################
+# DiffEq options
+###########################################################################################
+const DEFAULT_SDE_SOLVER = SOSRA() # default recommendation for additive noise
+const DEFAULT_STOCH_DIFFEQ = (alg=DEFAULT_SDE_SOLVER, DEFAULT_DIFFEQ_KWARGS...)
+
+# Function from user `@xlxs4`, see
+# https://github.com/JuliaDynamics/jl/pull/153
+function _decompose_into_sde_solver_and_remaining(diffeq)
+    if haskey(diffeq, :alg)
+        return (diffeq[:alg], _delete(diffeq, :alg))
+    else
+        return (DEFAULT_SDE_SOLVER, diffeq)
+    end
+end
+
+###########################################################################################
+# Type
+###########################################################################################
+
+# Define CoupledSDEs
+"""
+    CoupledSDEs <: ContinuousTimeDynamicalSystem
+    CoupledSDEs(f, g, u0 [, p, σ]; kwargs...)
+
+A stochastic continuous time dynamical system defined by a set of
+coupled ordinary differential equations as follows:
+```math
+d\\vec{u} = \\vec{f}(\\vec{u}, p, t) dt + \\vec{g}(\\vec{u}, p, t) dW_t
+```
+
+Optionally provide the overall noise strength `σ`, the parameter container `p` and initial time as keyword `t0`. If `σ` is provided, the diffusion function `g` is multiplied by `σ`.
+
+For construction instructions regarding `f, u0` see the [DynamicalSystems.jl tutorial](https://juliadynamics.github.io/DynamicalSystems.jl/latest/tutorial/#CoupledODEs).
+
+The stochastic part of the differential equation is defined by the function `g` and the keyword arguments `noise_rate_prototype` and `noise`. `noise` indicates the noise process applied during generation and defaults to Gaussian white noise. For details on defining various noise processes, refer to the noise process documentation page. `noise_rate_prototype` indicates the prototype type instance for the noise rates, i.e., the output of `g`. It can be any type which overloads `A_mul_B!` with itself being the middle argument. Commonly, this is a matrix or sparse matrix. If this is not given, it defaults to `nothing`, which means the problem should be interpreted as having diagonal noise.
```

Comment:
In general throughout the docstrings and the source code, ensure every line of code does not exceed 92 characters. Apply this principle throhgout the files you have added, and also ensure that you turn off line wrapping in your text editor.

Line breaks are a programmatically meaningful input so they shouldn't be visualized automatically by text editors.

Reply:
When you say "refer to the noise process documentation page", where does this refer? DifferentialEquations.jl?

Reply:
I formatted the docstrings. In general, I applied JuliaFormatter with blue style to the code. 

Re
>When you say "refer to the noise process documentation page", where does this refer? DifferentialEquations.jl?

I added a link to the DiffEqNoiseProcess documentation.